### ISSUE TITLE
Globally configure dirs through vim_npr_default_dirs in .vimrc

### ISSUE THREAD

**Issue by nkabbara** [tomarrell/vim-npr#5]
Hello,

Thanks for the great plugin. 

Is this the correct way to add 'src/screens' to the lookup path? When I do so in package.json it works fine, but most of my projects have 'src/screens' and I'm trying to configure it globally through .vimrc.

`let g:vim_npr_default_dirs = ["src", "lib", "test", "public", "node_modules", "src/screens"]`

**Comment by tomarrell** [comment]
Hey Nash,

This list of default dirs is only used if the resolve key is not found in the `package.json`.

It could be possible to add a separate list of paths to be appended to the resolve key list, which would allow you to apply this path globally.

I'm currently on holiday, but I'd be happy to merge a PR if you would like to work on it.

Cheers

**Comment by nkabbara** [comment]
Yup, I made sure the key isn’t there, but it still wasn’t taking. That’s
the behavior I want actually.

**Comment by tomarrell** [comment]
Hmm that seems odd. I’ll have to have a look. Would you be able to provide
an minimum reproducible example that I can test on?
